content/en/acknowledgements.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Acknowledgements
+menu: { main: { weight: 100 } }
+---
+
+{{% blocks/lead color="primary" %}}
+# Acknowledgements
+{{% /blocks/lead %}}
+
+{{% blocks/section type="section" color="white" %}}
+
+
+We would like to acknowledge the following sources for some of the content on this site:
```

Comment:
I have a key question: have we been granted permission to use content from those sources? If not, this can be a problem since _our_ docs are CC-by-4, but the original docs might not be. I suppose that if the source docs permit copying / use as long as there's attribution, then we're ok.

Either way, the prose here should probably make that clear (i.e., that we have permission to copy and/or make use of the docs from these sources).

Reply:
For the relevant sources that came from me (actually my employer), yes, there is permission to use that content. I also fully expect the content here to be further modified and integrated with other stuff with time.